packages/vow/src/watch-utils.js

```diff
@@ -206,11 +212,6 @@ export const prepareWatchUtils = (
               })
             : result;
 
-          const idToNonStorableResults = provideLazyMap(
-            utilsToNonStorableResults,
-            this.facets.utils,
-            () => new Map(),
-          );
           const nonStorableResults = provideLazyMap(
```

Comment:
`provideLazyMap` is defined to take a _weak_ map as its first parameter; this is an uncaught violation. But given the simple definition, it might as well expand from `@param {WeakMap} map` to `@param {Map | WeakMap} map`.

Reply:
WeakMap is a subset of Map, so not a violation

Reply:
[TypeScript Playground counterexample](https://www.typescriptlang.org/play/?filetype=js&useJavaScript=trueq=122#code/PQKhAIAEBcE8AcCm4DeAeA0gGgGoD4AKAWwC5wBZAQ3k1zwEpwBePcAOwFciAjRAJwC+4EMABQAYwD2bAM7RwMgJYAvRAHkAZs3BFmrIgDolqgNyiJ0ueAAe2togDu4AOqJKAayrwC9MxdmSADaIBoGSAOYExuoaBNb0vkA)
> ```
> Argument of type 'WeakMap<object, any>' is not assignable to parameter of type 'Map<object, any>'.
>   Type 'WeakMap<object, any>' is missing the following properties from type 'Map<object, any>': clear, forEach, size, entries, and 3 more.
> ```

Reply:
Confirmed; I was just wrong.